flink-java/src/main/java/org/apache/flink/api/java/operators/PartitionOperator.java

```diff
@@ -148,9 +147,6 @@ else if (pKeys instanceof Keys.SelectorFunctionKeys) {
 			}
 			
 		} 
-		else if (pMethod == PartitionMethod.RANGE) {
```

Comment:
Should we check this in some else place? I mean it's more like a sort issue, not a range partitioner issue. If  `pKeys.getKeyType().isSortKeyType() == false` and not fully sort, user just want to balance skew data through range partitioner shuffle, we should support it, right?